# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-07-02 15:37+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 3.6\n"
"X-Poedit-Basepath: ..\n"
"X-Poedit-SearchPath-0: findkerneldevice.py\n"
"X-Poedit-SearchPath-1: message.py\n"
"X-Poedit-SearchPath-2: plugin.py\n"
"X-Poedit-SearchPath-3: schermen.py\n"

#: message.py:35
msgid "No supported receiver found!"
msgstr ""

#: message.py:39
msgid "BACK-UP TOOL, FOR MAKING A COMPLETE BACK-UP"
msgstr ""

#: message.py:43
msgid "Please be patient, a backup will now be made, this will take about:"
msgstr ""

#: message.py:47
msgid " size to be backed up:"
msgstr ""

#: message.py:51
msgid "not found, the backup process will be aborted!"
msgstr ""

#: message.py:55
msgid "Some information about the task"
msgstr ""

#: message.py:59
msgid "Create: root.ubifs"
msgstr ""

#: message.py:63
msgid "Create: kerneldump"
msgstr ""

#: message.py:67
msgid "Additional backup ->"
msgstr ""

#: message.py:71
msgid "USB Image created in:"
msgstr ""

#: message.py:75
msgid "and there is made an extra copy in:"
msgstr ""

#: message.py:79
msgid "Please check the manual of the receiver on how to restore the image."
msgstr ""

#: message.py:83
msgid "Image creation FAILED!"
msgstr ""

#: message.py:87
msgid "available"
msgstr ""

#: message.py:91
msgid "There is a valid USB-flashdrive detected in one of the USB-ports, therefore an extra copy of the back-up image will now be copied to that USB-flashdrive."
msgstr ""

#: message.py:92
msgid "This only takes about 20 seconds....."
msgstr ""

#: message.py:96
msgid "Backup finished and copied to your USB-flashdrive."
msgstr ""

#: message.py:100
msgid "Full back-up to the harddisk"
msgstr ""

#: message.py:104
msgid "There is NO valid USB-stick found, so I've got nothing to do."
msgstr ""

#: message.py:106
msgid "PLEASE READ THIS:"
msgstr ""

#: message.py:107
msgid "To back-up directly to the USB-stick, the USB-stick MUST contain a file with the name:"
msgstr ""

#: message.py:108
msgid "backupstick or"
msgstr ""

#: message.py:109
msgid "backupstick.txt"
msgstr ""

#: message.py:111
msgid "If you place an USB-stick containing this file then the back-up will be automatically made onto the USB-stick and can be used to restore the current image if necessary."
msgstr ""

#: message.py:112
msgid "The program will exit now."
msgstr ""

#: message.py:116
msgid "Full back-up direct to USB"
msgstr ""

#: message.py:120
msgid "The content of the folder is:"
msgstr ""

#: message.py:124
msgid "Time required for this process:"
msgstr ""

#: message.py:128
msgid "minutes"
msgstr ""

#: message.py:132
msgid "Backup done with:"
msgstr ""

#: message.py:136
msgid "KB per second"
msgstr ""

#: message.py:140
msgid "Most likely this back-up can't be restored because of it's size, it's simply too big to restore. This is a limitation of the bootloader not of the back-up or the BackupSuite."
msgstr ""

#: message.py:144
msgid "There COULD be a problem with restoring this back-up because the size of the back-up comes close to the maximum size. This is a limitation of the bootloader not of the back-up or the BackupSuite."
msgstr ""

#: message.py:148
msgid "* * * WARNING * * *"
msgstr ""

#: message.py:149
msgid "Not enough free space on"
msgstr ""

#: message.py:153
msgid " to make a back-up!"
msgstr ""

#: message.py:157
msgid " MB available space"
msgstr ""

#: message.py:161
msgid " MB needed space"
msgstr ""

#: message.py:165
msgid "The program will abort, please try another medium with more free space to create your back-up."
msgstr ""

#: message.py:169
msgid "is not executable..."
msgstr ""

#: message.py:173
msgid "Using your own custom lookuptable.txt from the folder /etc"
msgstr ""

#: message.py:177
msgid "Version unknown, probably not installed the right way."
msgstr ""

#: message.py:181
msgid "not installed yet, now installing"
msgstr ""

#: message.py:185
msgid "Probably you are trying to make the back-up in flash memory"
msgstr ""

#: message.py:189
msgid "No additional USB-stick found to copy an extra backup"
msgstr ""

#: message.py:193
msgid "Installed packages contained in this backup:"
msgstr ""

#: message.py:197
msgid "NFI Image created in:"
msgstr ""

#: message.py:201
msgid "Mount point does not exist! Please check your network share configuration."
msgstr ""

#: message.py:205
msgid "Not a mounted filesystem! Please mount the network share first."
msgstr ""

#: message.py:209
msgid "Write permission denied! Check share permissions."
msgstr ""

#: message.py:213
msgid "Insufficient free space on NAS!"
msgstr ""

#: message.py:217
msgid "Unable to create backup directories!"
msgstr ""

#: message.py:221
msgid "Unable to create the backup marker file!"
msgstr ""

#: message.py:225
msgid "Full back-up to the MultiMediaCard"
msgstr ""

#: message.py:229
msgid "Backup started..."
msgstr ""

#: message.py:233
msgid "Phase 1/3: Preparing backup environment"
msgstr ""

#: message.py:237
msgid "Phase 2/3: Creating backup image"
msgstr ""

#: message.py:241
msgid "Phase 3/3: Finalizing backup"
msgstr ""

#: message.py:245
msgid "Backup completed successfully!"
msgstr ""

#: message.py:249
msgid "Backup statistics:"
msgstr ""

#: message.py:253
msgid "Backup size:"
msgstr ""

#: message.py:257
msgid "Backup progress:"
msgstr ""

#: message.py:261
msgid "Backup created in:"
msgstr ""

#: message.py:265
msgid "Dumping kernel (25%)"
msgstr ""

#: message.py:269
msgid "Creating root filesystem (50%)"
msgstr ""

#: message.py:273
msgid "Assembling image (75%)"
msgstr ""

#: message.py:277
msgid "Making extra copy (90%)"
msgstr ""

#: message.py:281
msgid "Finalizing (95%)"
msgstr ""

#: message.py:285
msgid "Backup complete (100%)"
msgstr ""

#: plugin.py:236 plugin.py:237
msgid "GB"
msgstr ""

#: plugin.py:239
msgid "Unknown"
msgstr ""

#: plugin.py:330 plugin.py:674
msgid "USB Storage"
msgstr ""

#: plugin.py:335 plugin.py:673
msgid "SD Card"
msgstr ""

#: plugin.py:340 plugin.py:675
msgid "Hard Disk"
msgstr ""

#: plugin.py:345 plugin.py:676
msgid "Network Storage"
msgstr ""

#: plugin.py:350 plugin.py:605 plugin.py:677
msgid "Barry Allen"
msgstr ""

#: plugin.py:355 plugin.py:615
msgid "Restore Backup"
msgstr ""

#: plugin.py:478
msgid "Make or restore a backup"
msgstr ""

#: plugin.py:480 plugin.py:836 plugin.py:1271
msgid "Close"
msgstr ""

#: plugin.py:481
msgid "Backup"
msgstr ""

#: plugin.py:482
msgid "Restore"
msgstr ""

#: plugin.py:483
msgid "Help"
msgstr ""

#: plugin.py:536
msgid ""
"Missing backup scripts:\n"
"\n"
"{}\n"
"\n"
"Please reinstall the BackupSuite plugin."
msgstr ""

#: plugin.py:577
msgid "Backup to {}"
msgstr ""

#: plugin.py:594
msgid "Backup to Network"
msgstr ""

#: plugin.py:686
msgid ""
"Do you want to make a backup to {}?\n"
"\n"
"This may take several minutes."
msgstr ""

#: plugin.py:706
msgid "Backup script not found for {}: {}"
msgstr ""

#: plugin.py:731
msgid "No network shares mounted! Please mount a share first."
msgstr ""

#: plugin.py:739
msgid "Free:"
msgstr ""

#: plugin.py:739
msgid "Total:"
msgstr ""

#: plugin.py:745
msgid "Select network share for backup:"
msgstr ""

#: plugin.py:759
msgid ""
"Could not create backup directory: {}\n"
"Error: {}"
msgstr ""

#: plugin.py:802
msgid "Backup failed: Not enough free space on device!"
msgstr ""

#: plugin.py:804
msgid "Backup failed: Write permission denied!"
msgstr ""

#: plugin.py:806
msgid "Backup failed: Device is read-only!"
msgstr ""

#: plugin.py:808
msgid "Backup failed: Required files not found!"
msgstr ""

#: plugin.py:810
msgid "Backup failed: Network connection lost!"
msgstr ""

#: plugin.py:814
msgid "Backup failed! Last errors:"
msgstr ""

#: plugin.py:818
msgid "Backup failed! Check log: {}"
msgstr ""

#: plugin.py:835
msgid "Select the folder with backup"
msgstr ""

#: plugin.py:840
msgid "current:  {}"
msgstr ""

#: plugin.py:875
msgid "Current: {}"
msgstr ""

#: plugin.py:878 plugin.py:1124
msgid "Unzip"
msgstr ""

#: plugin.py:881
msgid "Flash"
msgstr ""

#: plugin.py:883
msgid "About"
msgstr ""

#: plugin.py:884 plugin.py:1166
msgid "Delete"
msgstr ""

#: plugin.py:925
msgid ""
"\n"
"You are using dual multiboot!"
msgstr ""

#: plugin.py:928
msgid ""
"Warning!\n"
"Use at your own risk! Make always a backup before use!\n"
"Don't use it if you use multiple ubi volumes in ubi layer!"
msgstr ""

#: plugin.py:943
msgid "Run flash"
msgstr ""

#: plugin.py:949
msgid "Select parameter for start flash!\n"
msgstr ""

#: plugin.py:950
msgid "For flashing your receiver files are needed:\n"
msgstr ""

#: plugin.py:1022
msgid ""
"\n"
"The found files:"
msgstr ""

#: plugin.py:1025
msgid "  {} (maybe ok)"
msgstr ""

#: plugin.py:1028
msgid "  {} (maybe error)"
msgstr ""

#: plugin.py:1031
msgid " nothing!"
msgstr ""

#: plugin.py:1037 plugin.py:1071
msgid "Simulate (no write)"
msgstr ""

#: plugin.py:1038 plugin.py:1074
msgid "Standard (root and kernel)"
msgstr ""

#: plugin.py:1039 plugin.py:1080
msgid "Only root"
msgstr ""

#: plugin.py:1040 plugin.py:1083
msgid "Only kernel"
msgstr ""

#: plugin.py:1043 plugin.py:1086
msgid "Simulate second partition (no write)"
msgstr ""

#: plugin.py:1044 plugin.py:1089
msgid "Second partition (root and kernel)"
msgstr ""

#: plugin.py:1045 plugin.py:1092
msgid "Second partition (only root)"
msgstr ""

#: plugin.py:1046 plugin.py:1095
msgid "Second partition (only kernel)"
msgstr ""

#: plugin.py:1052
msgid "Exit"
msgstr ""

#: plugin.py:1063
msgid "A recording is currently running. Please stop the recording before trying to start a flashing."
msgstr ""

#: plugin.py:1068
msgid "Flashing: "
msgstr ""

#: plugin.py:1101
msgid "NOT found files for flashing!\n"
msgstr ""

#: plugin.py:1107
msgid "Show only found image and mtd partitions.\n"
msgstr ""

#: plugin.py:1112
msgid "ofgwrite will stop enigma2 now to run the flash.\n"
msgstr ""

#: plugin.py:1113
msgid "Your STB will freeze during the flashing process.\n"
msgstr ""

#: plugin.py:1114
msgid "Please: DO NOT reboot your STB and turn off the power.\n"
msgstr ""

#: plugin.py:1115
msgid "The image or kernel will be flashing and auto booted in few minutes.\n"
msgstr ""

#: plugin.py:1130
msgid "Do you really want to unpack {} ?"
msgstr ""

#: plugin.py:1133
msgid "Backup info"
msgstr ""

#: plugin.py:1170
msgid ""
"You are about to delete this backup:\n"
"\n"
"{}\n"
"Continue?"
msgstr ""

#: plugin.py:1181
msgid "Delete backup"
msgstr ""

#: plugin.py:1192
msgid "BackupSuite Help"
msgstr ""

#: plugin.py:1195
msgid ""
"BackupSuite v{}\n"
"\n"
msgstr ""

#: plugin.py:1196
msgid ""
"Welcome to BackupSuite help.\n"
"\n"
msgstr ""

#: plugin.py:1197
msgid "Device List:\n"
msgstr ""

#: plugin.py:1198
msgid "• Use UP/DOWN buttons to navigate through backup options\n"
msgstr ""

#: plugin.py:1199
msgid ""
"• Press OK to select an option\n"
"\n"
msgstr ""

#: plugin.py:1201
msgid "Backup Options:\n"
msgstr ""

#: plugin.py:1202
msgid "• HDD Backup: Full system backup to internal hard drive\n"
msgstr ""

#: plugin.py:1203
msgid "• USB Backup: Create a bootable USB recovery stick\n"
msgstr ""

#: plugin.py:1204
msgid "• MMC Backup: Backup to external storage (eMMC)\n"
msgstr ""

#: plugin.py:1205
msgid "• NET Backup: Backup to network storage (NAS/SMB)\n"
msgstr ""

#: plugin.py:1206
msgid "• Barry Allen: Multiboot backup (if installed)\n"
msgstr ""

#: plugin.py:1207
msgid ""
"• Restore: Flash a previously created backup or image\n"
"\n"
msgstr ""

#: plugin.py:1209
msgid "Button Functions:\n"
msgstr ""

#: plugin.py:1210
msgid "• GREEN: Start backup for selected device\n"
msgstr ""

#: plugin.py:1211
msgid "• YELLOW: Start restore process\n"
msgstr ""

#: plugin.py:1212
msgid "• BLUE: Show this help screen\n"
msgstr ""

#: plugin.py:1213
msgid ""
"• RED: Close BackupSuite\n"
"\n"
msgstr ""

#: plugin.py:1215
msgid "NET Backup Instructions:\n"
msgstr ""

#: plugin.py:1216
msgid "1. Mount your network share first:\n"
msgstr ""

#: plugin.py:1217
msgid "   - Go to Main Menu > Setup > System > Storage Manager\n"
msgstr ""

#: plugin.py:1218
msgid "   - Select 'Network Storage' and add your NAS/SMB share\n"
msgstr ""

#: plugin.py:1219
msgid "   - Enter server IP, share name, username and password\n"
msgstr ""

#: plugin.py:1220
msgid ""
"   - Mount the share and assign a name (e.g., NET_BACKUP)\n"
"\n"
msgstr ""

#: plugin.py:1221
msgid "2. Select 'NET Backup' from the device list\n"
msgstr ""

#: plugin.py:1222
msgid "3. Choose your mounted network share\n"
msgstr ""

#: plugin.py:1223
msgid ""
"4. Confirm and start the backup process\n"
"\n"
msgstr ""

#: plugin.py:1225
msgid "Best Practices:\n"
msgstr ""

#: plugin.py:1226
msgid "- Keep regular backups of your system\n"
msgstr ""

#: plugin.py:1227
msgid "- Verify you have enough free storage space\n"
msgstr ""

#: plugin.py:1228
msgid "- Never interrupt backup/restore processes\n"
msgstr ""

#: plugin.py:1229
msgid "- Use quality USB drives for recovery sticks\n"
msgstr ""

#: plugin.py:1230
msgid ""
"- For network backups, use wired connection instead of Wi-Fi\n"
"\n"
msgstr ""

#: plugin.py:1232
msgid "Technical Notes:\n"
msgstr ""

#: plugin.py:1233
msgid "- HDD backups: Stored in /media/hdd/backup\n"
msgstr ""

#: plugin.py:1234
msgid "- USB backups: Require FAT32-formatted drive with 'backupstick' file\n"
msgstr ""

#: plugin.py:1235
msgid "- NET backups: Require mounted CIFS/NFS share with write permissions\n"
msgstr ""

#: plugin.py:1236
msgid "- Restore: Works from HDD, USB, NET or MMC storage\n"
msgstr ""

#: plugin.py:1237
msgid ""
"- Always choose 'Standard (root and kernel)' when restoring\n"
"\n"
msgstr ""

#: plugin.py:1239
msgid "Troubleshooting NET Backup:\n"
msgstr ""

#: plugin.py:1240
msgid "- Ensure NAS is powered on and accessible\n"
msgstr ""

#: plugin.py:1241
msgid "- Verify username/password are correct\n"
msgstr ""

#: plugin.py:1242
msgid "- Check share permissions (read/write)\n"
msgstr ""

#: plugin.py:1243
msgid "- Test connection with ping command\n"
msgstr ""

#: plugin.py:1244
msgid ""
"- Use IP address instead of hostname if DNS fails\n"
"\n"
msgstr ""

#: plugin.py:1246
msgid "For detailed instructions and support:\n"
msgstr ""

#: plugin.py:1247
msgid "Visit our GitHub repository:\n"
msgstr ""

#: plugin.py:1248
msgid "https://github.com/persianpros/BackupSuite-PLi\n"
msgstr ""

#: plugin.py:1270
msgid "What is new since the last release?"
msgstr ""

#: plugin.py:1272
msgid "Please wait"
msgstr ""

#: plugin.py:1287
msgid "Change log not available"
msgstr ""

#: plugin.py:1308
msgid "Backup and restore your image"
msgstr ""
